sdk/python/packages/flet-video/src/flutter/flet_video/lib/src/video.dart

```diff
@@ -31,6 +32,24 @@ class _VideoControlState extends State<VideoControl> with FletStoreMixin {
   late final controller =
       VideoController(player, configuration: videoControllerConfiguration);
 
+  Future<void> _handleEnterFullscreen() async {
+    widget.control.updateProperties({"_fullscreen": true}, python: false);
```

Comment:
We have to re-think if we need these `python: false` at all.
I noticed in web mode if you do refresh that `_fullscreen` state, which is client-only, is lost. Right?